week2/homework/src/index.js

```diff
@@ -1,3 +1,97 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+const fileName = `${__dirname}/todoList.txt`;
+
+const command = process.argv[2];
+const args = process.argv.slice(3);
+
+const newItem = process.argv.slice(4);
+
+const number = parseInt(args);
+
+if (command === 'add') {
+  add(...args).then(() => console.log(`a new item has been added`));
+} else if (command === 'help' || command === undefined) {
+  help();
+} else if (command === 'list') {
+  list().then(data => console.log(`To-Dos:\n${data}`));
+} else if (command === 'remove') {
+  remove(number);
+} else if (command === 'reset') {
+  reset().then(() => console.log('All to-do items has been deleted'));
+} else if (command === 'update') {
+  update(number, newItem);
+}
+
+async function update(index, text) {
+  try {
+    const data = await list();
+    let arr = data.split('\n');
+    write(arr);
```

Comment:
Not needed. But in case you want to keep it, it should be `await`'ed, because `write` returns a promise.